test/recipes/controls/aws_parallelcluster_install/disable_services_spec.rb

```diff
@@ -0,0 +1,42 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'services_disabled_on_debian_family' do
+  title 'Test that DLAMI multi eni helper is disabled and masked on debian family'
+
+  only_if { os_properties.debian_family? && !os_properties.docker? }
```

Comment:
here you should use os_properties.virtualized? 
The param has been renamed as part of: https://github.com/aws/aws-parallelcluster-cookbook/pull/1709